components/Cards/card.css

```diff
@@ -0,0 +1,231 @@
+.cards-wrapper {
+    background-color:var(--white);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: row;
+    justify-self: center;
+    align-self: center;
+    padding: 2rem;
+    width: fit-content;
+    margin: 2rem;
+    flex-wrap: wrap;
+    position: relative;
+}
+
+.card,
+.card-basic {
+    width: 22rem;
+    height: 28rem;
+    background-color: var(--secondary-color);
+    border-radius: 0.5rem;
+    -webkit-border-radius: 0.5rem;
+    -moz-border-radius: 0.5rem;
+    -ms-border-radius: 0.5rem;
+    -o-border-radius: 0.5rem;
```

Comment:
As `border-radius` has full support now in all the browsers, you could avoid specifying it individually. Specifying the browser-oriented style comes in handy when some browser doesn't have full support for a property.

Reply:
It gets added automatically due to some VSCode extension. Removed it. Thanks for the suggestion